src/main/tray/tray.ts

```diff
@@ -62,7 +67,7 @@ export function initTray(windowManager: WindowManager) {
   const disposers = [
     autorun(() => {
       try {
-        const menu = createTrayMenu(windowManager);
+        const menu = createTrayMenu(windowManager, toJS(trayMenuItems.get()));
```

Comment:
Why do we need `toJS()` here? As `trayMenuItems` is observable box afaik and calling `.get()` already unwraps all observability == getting plain js/json? Wdyt?

Reply:
I created an extension that had `trayMenus` as an observable array and added a menu after the extension was loaded to test if the extension can change its menus and this `autorun` then went into infinite loop after the `trayMenus` changed. This fixed it. Not sure why, I'm not a `mobx` pro but maybe something to do with nested arrays?